### DISCUSSION TITLE
Showing Pull Requests and linked Issues

### DISCUSSION THREAD

**Discussion by rodsfreitas** [community/community#5535] [Projects and Issues] [answered]
The board view is fine, but when dealing with PR and linked issues, there is a bit of rework moving both.

It would be great to have the same functionality to show Pull Requests and linked issues as in the current `Project`. This would be available on the board primarily.

![image](https://user-images.githubusercontent.com/5301435/132606343-3c9ff799-d873-4931-933e-6b07ed35b88e.png)

What might be better to move the PR and the linked issues is to invert the order and show all the issues linked to the Pull Requests, for cases when a specific PR is closing multiple issues in its scope.

**Comment by azenMatt** [comment, marked as answer]
Hi @rodsfreitas - we just released this feature! 

You can check it out here - https://github.blog/changelog/2021-12-16-the-new-github-issues-12-16-update/

![changelog_-_linked_prs_v2 2c](https://user-images.githubusercontent.com/7584089/146577210-abb56ee5-37da-45eb-81a6-0b05f92fbb48.gif)
